packages/babel-core/test/api.js

```diff
@@ -574,6 +574,11 @@ describe("api", function() {
       it("builds external helpers in var output type", function() {
         babel.buildExternalHelpers(null, "var");
       });
+
+      it("injects external helpers in global context", function() {
+        babel.injectExternalHelpers();
+        assert.ok(global.babelHelpers);
```

Comment:
We should run this test in a new vm context, to avoid polluting the global namespace.

Reply:
can you guide me on how on how I should go about doing that?

Reply:
Moved the test to a different file. Now I think it should be using a new vm

Reply:
I don't know exactly how our test runner works, but I think that `global` is the same for every file. I think we should execute this test in a new `global` context, using the [vm](https://nodejs.org/api/vm.html) module.

Reply:
I asked in the jest discord channel. They confirmed that they use a new vm for each file. 
Also I tried using the vm module. Its not possible to test it that way because we can't import babel inside the sandbox so we have to pass babel from outside the sandbox. If we do it this way the babel that runs inside the sandbox will create the global outside. I tried this code
```js
const sandbox = { babel }; // this is babel imported at top of the file
const code = `
  babel.injectExternalHelpers();
`;
vm.createContext(sandbox);
vm.runInNewContext(code, sandbox);
assert.ok(sandbox.babelHelpers); //false
assert.ok(babelHelpers); // true
```

Reply:
👍

Reply:
👍